python/tank/authentication/errors.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, url):
         :param str url: Url of the site where login was attempted.
         """
         ShotgunAuthenticationError.__init__(
-            self, "Authentication using username/password is not allowed on the console for a SSO-enabled Shotgun site: %s" % url
+            self, "Authentication using username/password is not allowed on the console for a SSO-enabled (%s)" % url
```

Comment:
I think the original error was fine for that its worth. I would have simply removed the `: url` but, since its obvious from the UX which site you were logged into.

The new one however has a grammatical error.

Authentication using username/password is not allowed on the console for `an` SSO-enabled site.